cmd/podman/containers/create.go

```diff
@@ -188,6 +190,9 @@ func CreateInit(c *cobra.Command, vals entities.ContainerCreateOptions, isInfra
 			vals.UserNS = "private"
 		}
 	}
+	if registry.IsRemote() && cliVals.LogDriver != define.PassthroughLogging {
```

Comment:
```suggestion
	if registry.IsRemote() && cliVals.LogDriver == define.PassthroughLogging {
```
This should error for remote AND passthrough

Reply:
definitely :) fixed now